src/util/edit.cc

```diff
@@ -192,6 +196,41 @@ data::BinData EditEngine::getDataFromEditNode(const EditNode& edit_node,
   }
 }
 
+/*
+ * Warning: it can destruct iterator `it`
```

Comment:
This is not destruction but invalidation.

Reply:
DONE